src/EventDispatcher/AbstractGuzzleCacheEvent.php

```diff
@@ -0,0 +1,72 @@
+<?php
+namespace M6Web\Bundle\GuzzleHttpBundle\EventDispatcher;
+
+use Psr\Http\Message\RequestInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * Class AbstractGuzzleCacheEvent
+ */
```

Comment:
useless class comment

Reply:
Sadly we're using coke as a cs-fixer on this project 🙁